doc/ci.md

```diff
@@ -45,3 +45,11 @@ Since there is only one agent for PRs available, the last built PR wins.
 ### Deploy to Dev Environment
 
 The `master` branch will be automatically deployed to [dev.permaplant.net](https://dev.permaplant.net).
+
+### Deploy to Prod Environment
+
+There is a separate Jenkinsfile (`/ci/Jenkinsfile.release`) for production deployments. It can only be manually triggered inside Jenkins ([here](https://build.libelektra.org/job/PermaplanT-Release/)).
```

Comment:
```suggestion
There is a separate Jenkinsfile (`/ci/Jenkinsfile.release`) for production deployments.
It can only be manually triggered inside Jenkins ([here](https://build.libelektra.org/job/PermaplanT-Release/)).
```